### ISSUE TITLE
Bring on more devs

### ISSUE THREAD

**Issue by apbarratt** [rospogrigio/localtuya-homeassistant#1854]
LocalTuya is suffering from lack of maintenance and development.  @rospogrigio, would you be happy to assign a few extra people with full access to review issues and merge PRs and even publish releases?

We'd be bound to make mistakes whilst getting the hang of things, but it seems clear to me that some assistance in keeping this project alive would be beneficial :)

If so, how would such a team look in structure and numbers and communicate and agree on things?

**Comment by rospogrigio** [comment]
Yes, I wrote it many times, if anyone is willing to contribute and has the required skills they are more than welcome to step up and join the cause, I am still having too many personal issues to maintain the project properly.
Let me know, thanks

**Comment by findel** [comment]
> Yes, I wrote it many times, if anyone is willing to contribute and has the required skills they are more than welcome to step up and join the cause, I am still having too many personal issues to maintain the project properly. Let me know, thanks

Could you recommend a fork, or ask them to merge?

I've seen this one recommended by others.

https://github.com/xZetsubou/hass-localtuya

It qo be awesome not to have to uninstall this integration and install another. The work you did before was great!

**Comment by rospogrigio** [comment]
> I've seen this one recommended by others.
> 
> https://github.com/xZetsubou/hass-localtuya

Well, I have seen that fork too, and I have also tried it. It has progressed so much compared to this that I don't think they can be merged anymore, the documentation is very well done. However, I couldn't make it work on my appliance (I had removed my integration before trying, of course) but probably I just didn't spend enough time on it. Maybe you can give it a try and report feedback on your experience, what do you think?

**Comment by IgorMilavec** [comment]
I have just installed the localtuya integration and without #1836 I would not be able to configure it. At this time this integration is almost impossible to use for a regular user. This is why I believe it would be fair to clearly mark this integration as unmaintained/legacy/obsolete and refer users to a better integration.
I can't touch my setup during the holidays, but I will try switching to hass-localtuya and report back in mid January.

**Comment by IgorMilavec** [comment]
I have switched to hass-localtuya and I can confirm that it works fine for me. As it uses the same name as this repo, there are some caveats when switching, though. You need to delete the current integration, remove it from HACS, remove any templated entities (both from template integration, and then the entities themselves). After that you can download hass-localtuya and configure it. The problems I had due to not following this process are IMO due to a bug/design flaw in HACS and I'll open an issue there.
@rospogrigio if the current localtuya integration will not be actively maintained any more, could you please coordinate with the maintainer of hass-localtuya about the best migration path for most users?
